joblib/numpy_pickle.py

```diff
@@ -265,6 +265,12 @@ def save(self, obj):
             wrapper = self._create_array_wrapper(obj)
             Pickler.save(self, wrapper)
 
+            # A framer was introduced with pickle protocol 4 and we want to
+            # ensure the wrapper object is written before the numpy array
+            # buffer in the pickle byte stream.
```

Comment:
Is it not the pickle file rather than "pickle byte stream"?

Also maybe add a reference to https://www.python.org/dev/peps/pep-3154/#framing for completeness.